src/main/kotlin/org/jitsi/nlj/dtls/TlsClientImpl.kt

```diff
@@ -170,17 +171,34 @@ class TlsClientImpl(
         ProtocolVersion.DTLSv12.downTo(ProtocolVersion.DTLSv10)
 
     override fun notifyAlertRaised(alertLevel: Short, alertDescription: Short, message: String?, cause: Throwable?) {
-        val stack = with(StringBuffer()) {
-            val e = Exception()
-            for (el in e.stackTrace) {
-                appendln(el.toString())
+        when (alertDescription) {
+            AlertDescription.close_notify -> {
+                logger.info("close_notify raised, connection closing")
+            }
+            else -> {
+                logger.cerror {
+                    StringBuffer().apply {
+                        appendln("Alert raised: $alertLevel $alertDescription " +
+                            "(${AlertDescription.getName(alertDescription)})")
+                        val e = Exception()
+                        for (el in e.stackTrace) {
+                            appendln(el.toString())
+                        }
+                    }.toString()
+                }
```

Comment:
Do we want to do anything with `message` and `cause` if they're set, here?

Reply:
i remember originally logging those but must've gotten rid of them for some reason since they weren't logged now (always null maybe?).  does seem weird not to log them though.  i'll add them.